code/src/java/pcgen/gui2/facade/TempBonusHelper.java

```diff
@@ -159,20 +159,15 @@ public void commit()
 		{
 			return possibleEquipment;
 		}
-		boolean found = false;
 		theCharacter.setCalcEquipmentList(theCharacter.getUseTempMods());
-		for (Equipment aEq : charDisplay.getEquipmentSet())
+		FINDEQ: for (Equipment aEq : charDisplay.getEquipmentSet())
```

Comment:
Nit: Not a big fan of labels for control. Maybe I'm missing something but the previous approach was equivalent and easier to follow.